node/node.go

```diff
@@ -171,6 +184,21 @@ func NewNode(config *cfg.Config,
 	// reload the state (it may have been updated by the handshake)
 	state = sm.LoadState(stateDB)
 
+	/* TODO
+	// Generate node PrivKey
+	privKey := crypto.GenPrivKeyEd25519()
```

Comment:
I've been debating if this should be ephemeral.

In the ephemeral version, I imagine perhaps an ssh like confirm dialog in signatory before signatory starts accepting signing requests for a node.

if this passed in via a config file, then signatory would maintain a white list